src/sentry/integrations/slack/spec.py

```diff
@@ -9,11 +9,13 @@
 from sentry.models.notificationaction import ActionService
 from sentry.rules.actions import IntegrationEventAction
 
+PROVIDER = "slack"
```

Comment:
Mainly curious about this, but is there any chance someone can accidentally pollute their imports with this being identical between specs?

Reply:
Er, yes, very much so. 😅

Instead of referring to this constant, it might be safer in other places to just write `SlackMessagingSpec().provider_slug`, which is clearer but also more semantically noisy. (It looks weird at first glance to instantiate the class just to retrieve a single property. There's an argument for making `provider_slug` a class variable, but then we couldn't annotate it as an `@abstactmethod` in the same way.)